sql-parser/src/main/java/io/crate/sql/tree/DropFunction.java

```diff
@@ -92,5 +92,12 @@ public PrivilegeType privilegeType() {
     }
 
     @Override
-    public String clazz() { return "SCHEMA";}
+    public PrivilegeClazz privilegeClazz() {
+        return PrivilegeClazz.SCHEMA;
+    }
+
+    @Override
+    public String privilegeIdent() {
+        return name.getParts().get(0);
```

Comment:
also here, schema is always the first part? its not if the schema can be omitted...